src/sia/services/response_handler.py

```diff
@@ -126,7 +130,7 @@ def process_query(
         factory: Factory,
         params: SIAv2Parameters,
         sia_query: SIAv2QueryType,
-        request: Request,
+        context: RequestContext,
```

Comment:
I've been explicitly passing the events object (`context.events` here) to the service constructors (or class methods in this case) instead of passing the whole request context or process context (based on [this recommendation](https://sqr-072.lsst.io/#process-context), so that it's easy(er) to see at a glance what a service might need or do.

I don't totally buy that we should be this explicit with stuff like the events object, which theoretically could be used in any code anywhere, but I haven't convinced anyone else yet :smiley:

Reply:
Oh right ok, I've modified this to pass in the events object to the method here instead.